src/main/kotlin/notary/endpoint/eth/EthRefundStrategyImpl.kt

```diff
@@ -78,13 +84,17 @@ class EthRefundStrategyImpl(
                     val destEthAddress = ""
                     logger.info { "Rollback case ($key, $value)" }
 
-                    EthRefund(destEthAddress, "mockCoinType", "10", request.irohaTx)
+                    val relayAddress = relayProvider.getRelays().get().filter {
```

Comment:
I think get() is not appropriate function here. I'd rather use map() or fold() instead